packages/navigation-gizmo/src/NavigationGizmo.ts

```
@@ -152,12 +156,19 @@ export class NavigationGizmo extends Script {
 
   override onUpdate() {
     this._gizmoCamera.viewport.set(this.position.x, this.position.y, this.size.x, this.size.y);
+
+    const matrix = NavigationGizmo._tempMat
```

ESLint

[error] Insert `;`

(prettier/prettier)